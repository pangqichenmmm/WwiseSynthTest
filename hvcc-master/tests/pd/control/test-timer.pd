#N canvas 628 201 198 318 10;
#X obj 41 241 timer;
#X obj 41 31 loadbang;
#X obj 41 61 t b b;
#X obj 41 271 print;
#X obj 41 91 del 69;
#X obj 41 121 t b b;
#X obj 68 151 del 145;
#X obj 68 181 t b b;
#X obj 95 211 del 131;
#X connect 0 0 3 0;
#X connect 1 0 2 0;
#X connect 2 0 4 0;
#X connect 2 1 0 0;
#X connect 4 0 5 0;
#X connect 5 0 0 1;
#X connect 5 1 6 0;
#X connect 6 0 7 0;
#X connect 7 0 0 1;
#X connect 7 1 8 0;
#X connect 8 0 0 1;
