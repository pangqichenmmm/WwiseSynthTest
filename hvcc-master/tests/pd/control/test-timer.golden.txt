[@ 69.000] print: 69
[@ 214.000] print: 214
[@ 345.000] print: 345
